lock/src/main/java/org/infinispan/lock/logging/Log.java

```diff
@@ -37,4 +37,10 @@
 
    @Message(value = "Invalid scope for tag <clustered-lock>. Expected CACHE_CONTAINER but was %s", id = 29007)
    ClusteredLockException invalidScope(String scope);
+
+   @Message(value = "Cannot create clustered locks when clustering is not enabled", id = 29008)
+   ClusteredLockException requireClustered();
+
+   @Message(value = "Configuration is not clustered, clustered locks are disabled", id = 29009)
+   void configurationNotClustered();
```

Comment:
Sorry not sure how I missed this, but this line is missing the `@LogMessage(level = INFO)` on it.